aws/logs_monitoring/tools/integration_tests/external_lambda/serverless.yml

```diff
@@ -1,7 +1,7 @@
 service: forwarder-tests-external-lambda
 provider:
   name: aws
-  runtime: python2.7
+  runtime: python3.11
```

Comment:
is this file really used ? I'm surprised in was on 2.X

Reply:
caught that when I was doing the version changes, not sure if it is used.